pages/osx/krunvm.md

```diff
@@ -0,0 +1,24 @@
+# krunvm
+
+> CLI-based utility for creating microVMs from OCI images.
+> More information: <https://github.com/containers/krunvm>.
+
+- Create MicroVM based on Fedora:
+
+`krunvm create docker.io/fedora --cpus {{number of vcpus}} --mem {{Memory in mbs}} --name {{name}}`
```

Comment:
```suggestion
`krunvm create {{docker.io/fedora}} --cpus {{number_of_vcpus}} --mem {{memory_in_mbs}} --name "{{name}}"`
```

- Optional fix: first placeholder can be enclosed to `{{` and `}}` as this positional argument accepts not only `docker.io/fedora` value.
- Mandatory fix: [placeholder format violation](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#naming): use [snake_case](https://wikipedia.org/wiki/snake_case) for multi-word placeholders.
- Optional fix: last placeholder can be quoted if it can contain any special characters for POSIX-like shells.